cypress/README.md

```diff
@@ -0,0 +1,40 @@
+# *WIP*
+
+## Concepts:
```

Comment:
can we describe methods and features, etc?

Reply:
I'm a bit hesitant on this one.
How far should I go into explaining BDD and Cypress.
As I see it, it's better to spend time to look and the official docs, which I linked, and consult our tests for examples.
If I add descriptions here, it's just duplication imo, which also get outdated when the underlying frameworks change.

Reply:
I'm only looking for very basic descriptions.

Reply:
I'm only looking for very basic descriptions.  Can go in the top level test readme.

Reply:
I thought I did this via 65e1f54.
What is still missing?

Reply:
Looks specifically for a quick summary about "methods" and "features".  A quick enumeration of the different file types (extensions) would be helpful too.  We need a quick sense of the steps involved in creating a test.  Nothing detailed, just very high level.

Reply:
> top level test readme

I hope we both understand /cypress for this piece of documentation

> high level

Done via 9fa56a4